Yoast/wordpress-seo
# Replace yoast.com/kb.yoast.com URLs with shortlinks

## Summary

This PR can be summarized in the following changelog entry:

* Replaces any Yoast domain URLs with shortlink alternatives.

## Technical choices

* Replaced `yoast.com` to `example.com` in unit tests. The domain does not matter, and this makes it clearer that those are not actual `yoast.com` references.

## Test instructions

This PR can be tested by following these steps:

* Open each shortlink to see if it matches the page that was previously used
  * Mind that there are pages that did not exist anymore, so new ones have been selected
* Make sure all links in the replaced locations work as expected

Fixes https://github.com/Yoast/wordpress-seo/issues/8051
